content/en/security/application_security/software_composition_analysis/_index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Software Composition Analysis
+kind: documentation
+aliases:
+  - /security/application_security/risk_management/
+  - /security/application_security/vulnerability_management/
+further_reading:
+- link: "/getting_started/application_security/software_composition_analysis"
+  tag: "Guide"
+  text: "Getting started with Software Composition Analysis"
+- link: "https://www.datadoghq.com/blog/iast-datadog-application-vulnerability-management/"
+  tag: "Blog"
+  text: "Enhance application security in production with Application Vulnerability Management"  
+- link: "/security/application_security/code_security"
+  tag: "documentation"
+  text: "Enable code security vulnerability detection on your services"
+- link: "/code_analysis/software_composition_analysis/"
+  tag: "documentation"
+  text: "Setup Software Composition Analysis on your CI pipelines"
+algolia:
+  tags: ['Software Composition Analysis', 'Vulnerability Management', 'SCA', 'AVM']
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Application Security Management is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+## Overview
+
+Datadog Software Composition Analysis (SCA) helps you leverage open source with confidence. The capabilities of SCA include vulnerability detection, business risk (library inventory and licensing information), and quality evaluation of the open source libraries in your services. The key differentiation factor powering Datadog SCA is the end-to-end coverage of your software development lifecycle: from the code that your developers commit, to the production applications already running in your Datadog deployment.
+
+Check [ASM Compatibility][6] to see if your service is supported.
+
+## Library Inventory
+
+The Datadog SCA Library Inventory helps you understand the list of libraries and its versions that compose your application. To access the library explorer, navigate to [Security > Application Security > Catalog > Library Explorer][8].
+Since Datadog SCA covers your software development life cycle end-to-end, the libraries are detected throughout the entire lifecycle of the application. The library inventory contains everything you need to know about the libraries, including name and version, and other risk aspects such as licenses, and quality aspects. 
```

Comment:
```suggestion
The Datadog SCA Library Inventory helps you understand the list of libraries and its versions that compose your application. To access the Library Explorer, navigate to [**Security** > **Application Security** > **Catalog** > **Library Explorer**][8].

Since Datadog SCA covers your software development life cycle end-to-end, the libraries are detected throughout the entire lifecycle of the application. The library inventory contains everything you need to know about the libraries, including name and version, and other risk aspects such as licenses and quality aspects. 
```